Files/MultilingualResources/Files.hu-HU.xlf

```diff
@@ -2078,11 +2072,11 @@
         </trans-unit>
         <trans-unit id="BundlesWidgetAddBundleErrorAlreadyExists" translate="yes" xml:space="preserve">
           <source>Bundle with the same name already exists!</source>
-          <target state="new">Bundle with the same name already exists!</target>
+          <target state="translated"Csomag ezzel a névvel már létezik!</target>
```

Comment:
Missing ">" here

![Captura de pantalla 2021-01-29 182502](https://user-images.githubusercontent.com/61259627/106329072-9966d300-625f-11eb-8c9e-c3e35e0861b2.png)

Reply:
Oops. Sorry I fixed it.